src/components/Dropdown/LanguageDropdown/LanguageDropdown.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useContext, useState } from 'react'
+import * as Styled from './styles'
+import Globe from '../../../../public/icons/globe.svg'
+import { LocaleContext } from '../../../contexts/LocaleContext'
+
+export default function LanguageDropdown() {
+  const { setLocale } = useContext(LocaleContext)
+  const [currentLocale, setCurrentLocale] = useState('English')
```

Comment:
Use locale/setLocale from the LocaleContext instead. Or is there a point in having this state inbetween?

Reply:
The context locale is only returning en or es-ES so to be able to display the current locale in the selector as English or Espanol I need to set a new variable based on what the user has selected.

Or am I missing something?

Reply:
Hm, alright! Wouldn't it be possible to use the context's values directly, and then on r31 use `localeToLanguage`? So all values always are in the locale-format, but the options spell out the language.